test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -1188,28 +529,8 @@ public async Task GetTrustResultAsync_WithTrustedButExpiredPrimaryAndTimestampCe
                 Assert.Equal(4, matchingIssues.Count);
                 SigningTestUtility.AssertOfflineRevocationOnlineMode(matchingIssues, LogLevel.Warning);
             }
-            SigningTestUtility.AssertRevocationStatusUnknown(matchingIssues, LogLevel.Warning, NuGetLogCode.NU3018);
-        }
-
-        [CIOnlyFact]
-        public async Task GetTrustResultAsync_WithTimestampChainingToUntrustedRoot_NotAllowIgnoreTimestamp_FailAsync()
```

Comment:
May I know why this test is removed? Thanks.

Reply:
It was renamed and moved.

The test wasn't testing `GetTrustResultAsync(...)` but `VerifySignaturesAsync(...)`.  I renamed the test to `VerifySignaturesAsync_WithTimestampChainingToUntrustedRoot_NotAllowIgnoreTimestamp_FailAsync` and moved it [here](https://github.com/NuGet/NuGet.Client/pull/3952/files#diff-6a15bb307e86dc7012938479f8a3fe9aff77788c02854afca190d0a2e5b4e405R749).